vite.config.ts

```diff
@@ -10,13 +10,13 @@ export default ({ mode }) => {
     base: BASE_URL,
     plugins: [react()],
     build: {
-      assetsDir: "assets",
+      assetsDir: "./assets",
       outDir: "dist",
       rollupOptions: {
         output: {
-          entryFileNames: `SMC-interactive-game.js`,
-          chunkFileNames: `SMC-interactive-game.js`,
-          assetFileNames: `SMC-interactive-game.[ext]`,
+          entryFileNames: `SMC-interactive-game-[hash].js`,

```

Comment:
We are going to have to think of a different way to hash these files as there is an external component referencing these files, and if the hash changes every build then the reference will break. maybe we can add a version query hash on our reference.

Reply:
I've updated so instead of hash we use the version specified in the package.json